src/models/onnxruntime_api.h

```diff
@@ -213,32 +213,7 @@ inline void InitApi() {
 
 #if !defined(__ANDROID__)
   if (ort_lib_handle == nullptr) {
-    const std::array<std::string, 4> target_libraries = {
-        std::string("libonnxruntime.so"),
-        std::string("libonnxruntime.so.1.18.0"),
-        std::string("libonnxruntime.so.1.19.0"),
-        std::string("libonnxruntime.so.1.20.0")};
-
-    // Search parent directory
-    std::string current_module_dir = GetCurrentModuleDir();
-    for (const std::string& lib_name : target_libraries) {
-      std::string pip_path{current_module_dir + "/" + lib_name};
-      ort_lib_handle = LoadDynamicLibraryIfExists(pip_path);
-      if (ort_lib_handle != nullptr) {
-        break;
-      }
-    }
-
-    if (ort_lib_handle == nullptr) {
-      // Search for pip installation
-      for (const std::string& lib_name : target_libraries) {
-        std::string pip_path{current_module_dir + "/../onnxruntime/capi/" + lib_name};
-        ort_lib_handle = LoadDynamicLibraryIfExists(pip_path);
-        if (ort_lib_handle != nullptr) {
-          break;
-        }
-      }
-    }
+    ort_lib_handle = LoadDynamicLibraryIfExists("libonnxruntime.so.1");
```

Comment:
Is this going to work with older ORT installation, or do we want to force ORT 1.19 with ort-genai 0.0.4?

Reply:
I don't think we need to make it work for older ort installation considering that the library depends on certain features only available in 1.19.0.
Moreover, --in the case of python-- the shared library is not packaged inside ort package before 1.19.0.